java/java-tests/testData/inspection/redundantStringOperation/BAOStoString.java

```diff
@@ -12,7 +12,7 @@ void charset() {
     Charset charset = Charset.defaultCharset();
 
     ByteArrayOutputStream baos = new ByteArrayOutputStream();
-    String s = new String(baos.toByteArray(), charset);
+    String s = new <warning descr="Inefficient conversion from ByteArrayOutputStream">String</warning>(baos.toByteArray(), charset);
```

Comment:
You are breaking this test. It was intentionally made for Java 9, see https://youtrack.jetbrains.com/issue/IDEA-258683. It should be executed with Java 9 language level.

Reply:
Reverted